modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -186,6 +183,42 @@ include::partial$include-reauthentication-conditions.adoc[]
 
 Connected apps and clients are exempt from reauthentication. If you have a script that makes an API call to change your email address, it might need to accommodate reauthentication.
 
+=== Verify Your Email Address
+
+If you have not verified your email address that you use to log in to Anypoint Platform, a prompt appears to verify your email when you log in.
+You can verify your email using the login prompt, or you can verify your email in your user profile.
+SSO users cannot verify their email addresses through Anypoint Platform and do not receive verification prompts.
+
+include::partial$include-nav-steps-to-profile.adoc[]
+
+. Click the *Contact Info* tab.
+. In the *Email* section, click *Verify Email*. +
+A window appears, prompting you to enter a verification code that you received at your Anypoint Platform account's email address.
```

Comment:
Instead of "received", should we say "will receive"? Sometimes it may take a minute to get the email with the verification code. The user may get confused as they will look for the code right after clicking on "Verify Email".